tests/unit/test_ingest.py

```diff
@@ -13,9 +13,10 @@ class testIngest(unittest.TestCase):
     Testing ingestion functionalities.
     """
 
+    @mock.patch('lega.ingest.getattr')
```

Comment:
Are you sure you want to patch it that way? The `getattr` is not exactly in the `lega.ingest` namespace.

Reply:
any suggestions to patch it another way ?

It is a global function, but it is used in that module, we do the same for `consume` and `get_connection`. I do recognize a more appropriate way to do it would be to mock the result for each `getattr`, but I think this is for a later PR when we revisit the unit-tests.